### ISSUE TITLE
[Service Health - History]: Screen reader announces twice on 'Filter' control list.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13205]
### Steps to reproduce:
1.	Open URL and login with valid credentials.
2.	Navigate to 'Health - Service Health - History'.
3.	Verify that screen reader announces twice on 'Filter' control list or not.

### Actual behavior:
Screen reader announces twice on 'Filter' control list "Microsoft 365 suite".

### Expected behavior:
Screen reader should not announce twice on 'Filter' control list "Microsoft 365 suite".

### Additional Notes:
1.	Same issue repro on 'Past 7 days' control list in Service Health->History-->Past 7 days control list.

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, also reproducible through NVDA version 2020.4

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
